pkg/apiserver/registry/stats/nodelatencystats/rest.go

```diff
@@ -50,38 +57,112 @@ func (r *REST) Destroy() {
 }
 
 func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
-	// TODO: fill this function in next PR
-	return &statsv1alpha1.NodeLatencyStats{}, nil
+	// Update will add the object if the key does not exist.
+	summary := obj.(*statsv1alpha1.NodeLatencyStats)
+	if err := r.indexer.Update(summary); err != nil {
+		return nil, errors.NewInternalError(err)
+	}
+
+	return summary, nil
 }
 
 func (r *REST) Get(ctx context.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
-	// TODO: fill this function in next PR
-	return &statsv1alpha1.NodeLatencyStats{}, nil
+	obj, exists, err := r.indexer.GetByKey(name)
+	if err != nil {
+		return nil, errors.NewInternalError(err)
+	}
+	if !exists {
+		return nil, errors.NewNotFound(statsv1alpha1.Resource("nodelatencystats"), name)
+	}
+
+	return obj.(*statsv1alpha1.NodeLatencyStats), nil
 }
 
 func (r *REST) NewList() runtime.Object {
 	return &statsv1alpha1.NodeLatencyStats{}
 }
 
 func (r *REST) List(ctx context.Context, options *internalversion.ListOptions) (runtime.Object, error) {
-	// TODO: fill this function in next PR
-	return &statsv1alpha1.NodeLatencyStatsList{}, nil
+	objs := r.indexer.List()
+
+	// Due to the unordered nature of map iteration and the complexity of controlling 'continue',
+	// we will ignore paging here and plan to implement it in the future.
+
+	entries := make([]statsv1alpha1.NodeLatencyStats, 0, len(objs))
+	for _, obj := range objs {
+		entries = append(entries, *obj.(*statsv1alpha1.NodeLatencyStats))
+	}
+
+	return &statsv1alpha1.NodeLatencyStatsList{Items: entries}, nil
 }
 
 func (r *REST) ConvertToTable(ctx context.Context, obj runtime.Object, tableOptions runtime.Object) (*metav1.Table, error) {
-	// TODO: fill this function in next PR
 	table := &metav1.Table{
 		ColumnDefinitions: []metav1.TableColumnDefinition{
-			{Name: "SourceNodeName", Type: "string", Format: "name", Description: "Source node name."},
-			{Name: "NodeLatencyStatsList", Type: "array", Format: "string", Description: "Node IP latency list."},
+			{Name: "Node Name", Type: "string", Format: "name", Description: "Name of Node from which latency was measured."},
+			{Name: "Num Latency Entries", Type: "integer", Format: "int64", Description: "Number of peers for which latency measurements are available."},
+			{Name: "Max Latency", Type: "integer", Format: "int64", Description: "Name of the peer with the highest latency and the latency value(ns)."},
+			{Name: "Avg Latency", Type: "integer", Format: "int64", Description: "Average latency value across all peers(ns)."},
```

Comment:
It may be more user-friendly to make this a string.
To convert from the integral latency value (in ns) to the string, you would use: `time.Duration(latency).String()`
This way the column will be displayed as a value with a user-friendly unit (e.g., `10ns`, `10µs`, `10ms`).
With this change, you can remove the unit specification (`(ns)`) from the column description.

Reply:
Ok, using formatted timestamp string is a better choice.